lib/bindConnectors/getConfigs.js

```diff
@@ -0,0 +1,29 @@
+const AWS = require('aws-sdk');
+
+const getCurrentRegion = () => {
+	return process.env.AWS_REGION === 'dev'
```

Comment:
At what point is `AWS_REGION` ever dev?

Reply:
this way of doing things was inspired by the csv connector code, but you are right it's confusing so we removed it